EIPS/eip-1.md

```diff
@@ -13,6 +13,26 @@ updated: 2015-12-07, 2016-02-01, 2018-03-21, 2018-05-29, 2018-10-17, 2019-05-19,
 
 EIP stands for Ethereum Improvement Proposal. An EIP is a design document providing information to the Ethereum community, or describing a new feature for Ethereum or its processes or environment. The EIP should provide a concise technical specification of the feature and a rationale for the feature. The EIP author is responsible for building consensus within the community and documenting dissenting opinions.
 
+## Table of Contents
+
+- EIP Submissions
+   - [EIP Design Rational](#eip-rationale)
+   - [EIP Types](#eip-types)
+   - [EIP Work Flow](#eip-work-flow) 
+ - EIP Format and Style Guide
+   - [Requirements](#what-belongs-in-a-successful-eip)
+   - Format
+     - [Template](#eip-formats-and-templates)
+     - [Header](#eip-header-preamble)
+   - [Auxiliary Files](#auxiliary-files)
+ - EIP Best Practices
+   - [Transfering Ownership](#transferring-eip-ownership)
+ - [EIP Editors](#eip-editors)
+ - [Hardfork Meta](#hardfork-meta)
+   - [Ethereum Hardforks](#ethereum-hardforks) 
```

Comment:
Do we want to call them network upgrades?

Reply:
Yeah my opinion is to call them network upgrades.

Reply:
Upgrade implies the new version is better and the old version is worse. Who decides which is better? That is centralization.

Reply:
Not sure I agree. IMO "upgrade" stands for "newer", rather than "better". Lots of examples in traditional software of people "not upgrading because the newer version is worse". 

OTOH, "hard fork" implies that there will be a fork, which, for non contentious upgrades, isn't the case. I would phrase the general case as "a network upgrade, which can result in a hard fork if a subset of the community doesn't update their software". 

FWIW, "network upgrade" is what is currently used in communications from the EF, see [Muir Glacier](https://blog.ethereum.org/2019/12/23/ethereum-muir-glacier-upgrade-announcement/), [Istanbul](https://blog.ethereum.org/2019/11/20/ethereum-istanbul-upgrade-announcement/), [Constantinople](https://blog.ethereum.org/2019/02/22/ethereum-constantinople-st-petersburg-upgrade-announcement/). Last use of "hard fork" was for [Byzantium](https://blog.ethereum.org/2017/10/12/byzantium-hf-announcement/).

Reply:
I'm neutral on this. Technically it is a hard fork, but I can also see that the community has preconceptions about what a hardfork means and it certainly isn't what most people think. To be a hardfork that results in a network split there is a lot of requirements. Contentious hardforks should be called network splits. As far as to call them Network Upgrades vs Hardforks I am open too either.